content/docs/command-reference/import.md

```diff
@@ -91,13 +91,32 @@ from the source repo.
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
+- `-b`, `--backup` - turns _backup mode_ on for this import, so it can be pushed
+  to remote storage (see [Backup mode](#backup-mode) for details).
+
 - `-h`, `--help` - prints the usage/help message, and exit.
 
 - `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
   problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Backup mode
+
+To avoid duplicating large files, `dvc import` does not push imported files to
+any [remotes](/doc/command-reference/remote) by default. This means that when
+`dvc pull/fetch` is run, DVC retrieves imports from their source.
```

Comment:
> when `dvc pull/fetch` is run, DVC retrieves imports from their source.

I didn't know that. Are you sure? I think you have to `dvc repro`. Please try it out 🙂

Reply:
It does! That was the motivation for this change actually, since I have a lot of git imports and all the repos were getting cloned each time I ran `dvc pull`. As a test, I ran `dvc pull -v` in a repo with a single import in it, like this:

```
md5: ca4361da725ee43b1e01ffe75f96b42a
frozen: true
deps:
- path: file.txt
  repo:
    url: ../imp
    rev_lock: f7d942abf891b4a0c29f8dec9e6728870862b012
outs:
- md5: f701279bc9ac499afb1f556e22c23cb2
  path: file.txt
```

Even after the import and the cache were present and up to date, the output contains

```
...
2020-09-20 22:20:08,091 DEBUG: Collecting information from local cache...
2020-09-20 22:20:08,094 DEBUG: Creating external repo ../imp@f7d942abf891b4a0c29f8dec9e6728870862b012                                                                                
2020-09-20 22:20:08,095 DEBUG: erepo: git clone '../imp' to a temporary dir
2020-09-20 22:20:08,192 DEBUG: Saving '../../../../private/var/folders/s8/gg_nx8mn23sbqxbj1021tn6c0000gn/T/tmpbg9ac34ldvc-clone/file.txt' to '.dvc/cache/f7/01279bc9ac499afb1f556e22c23cb2'.
...
```

Reply:
Interesting. I should know this 😅 but also `repro` will do it so we just need to mention that one too in the sentence, please.

I'll make a separate issue for us to mention that fetch and pull re-import data...

Reply:
> Even after the import and the cache were present and up to date (the repo is cloned)

On this, did DVC actually re-download the data? OR just clone the repo to check what the latest version was? Because if the former case, it sound like a bug to report in the core repo (shouldn't re-download imports if you already have the latest version).

Reply:
I think just clone to check but, in the case of git imports, that's the same since you need to redownload the entire git repo to clone it.

Reply:
Hmmm. I think it could/should just clone the bare repo and check the latest revision where the file was modified (blame) is <= the `rev` in the .dvc file (`deps.repo.rev` field, to be precise.) Anyway... Do you see any more -v output that indicates whether the file was actually pulled/checked-out from the Git remote? It may be hard to determine, actually. I can take a look later if not so don;t worry too much about this! 🙂 Thanks!

Reply:
So yes, `dvc pull` does download and overwrite the existing cached file even if it's the latest version (sounds like a bug to me, opened iterative/dvc/issues/4597). But only for Git imports, I think.

Anyway, resolving here!